src/atoms/Typography.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+
+import { Heading, SubHeading, Body, Code } from './Typography';
+
+<Meta title="atoms/Typography" parameters={{ actions: { argTypesRegex: '^on.*' } }} />
+
+# Typography
```

Comment:
Can we export `Text` as well.
It's nice to have a truly versatile typography component.

Reply:
Text is already exported from Rebass, but I have added it to the story 😄